win/CS/HandBrakeWPF/Helpers/AutoNameHelper.cs

```diff
@@ -97,8 +127,10 @@ public static string AutoName(EncodeTask task, string sourceOrLabelName, Preset
                         destinationFilename.Replace("{source}", sourceName)
                             .Replace(Constants.Title, dvdTitle)
                             .Replace(Constants.Chapters, combinedChapterTag)
-                            .Replace(Constants.Date, DateTime.Now.Date.ToShortDateString().Replace('/', '-'))
-                            .Replace(Constants.Time, DateTime.Now.ToString("HH:mm"));
+                            .Replace(Constants.Date, DateTime.Now.Date.ToString("MM-dd-yy"))

```

Comment:
Any reason you changed these to a specific format?  ShortDate is probably more appropriate

Reply:
Just personal preference. I originally thought this conveyed content better and removed the need to worry about the replace method. Thinking more about it though I did break consistency which is more important.

- [ ] modify statements to read like others in this class